src/hooks/useForm.js

```diff
@@ -15,21 +15,21 @@ const useForm = (cb, validate) => {
     }
   }, [error, isSubmitting])
 
-  const handleSubmit = useCallback(e => {
```

Comment:
Why `useCallback` has been removed?

Reply:
in our case, without second parameter this hook does not have sense.